frontend/src/App.js

```diff
@@ -167,9 +168,9 @@ class App extends React.Component {
 
   render() {
     // login prep
-    axios.get(`http://localhost:8080/state`).then(data => {
+    /*axios.get(`http://localhost:8080/state`).then(data => {
```

Comment:
nit: When you comment out code, I recommend you add an explanation for why it is commented out. Or better, just delete it. Gets confusing when you have a bunch of commented out code.